src/index.js

```diff
@@ -80,8 +80,23 @@ exports.ensureAuthenticated = (request, response, next) => {
 
 let alreadyInitialized = false;
 
+exports.initServices = (Implementation) => {
+  const { opts } = Implementation;
+  const integrator = new Integrator(opts, Implementation);
```

Comment:
Why isn't it done after the `requireAllModels` like at line 184?

Reply:
I have now use the same code than before expect it's correctly encapsulated. Didn't change the order of functions